Verifies fields towards the model. Requires Java 8.

ckEriksen opened the pull request:
If the field has been renamed, and the import uses the label as seen i Ardoq GUI, the import now uses the correct field names.

ckEriksen committed:
Verifies fields towards the model. Requires Java 8.

magnulf commented on src/main/java/com/ardoq/ExcelImport.java:
```diff
@@ -69,6 +71,59 @@ public static void main(String[] args) throws IOException {
 
+    private static void verifyModel() {
+        ArdoqClient client = new ArdoqClient(host, token);
+        client.setLogLevel(RestAdapter.LogLevel.NONE);
```
Ideally, setting the loglevel should be an option, and at least placed somewhere else, as it's hidden in verify model method now.

magnulf commented:
Nice - thank you! If you could place the log-level in the main method, it's good enough (tm) :-)

ckEriksen commented:
Updated the reference for the client to the last version, which includes corrections to the log level. Removed the specific setting.
Ammended the changes to the previous commit.
